Code/Tools/Libs/GuiFoundation/DockPanels/ApplicationPanel.moc.h

```diff
@@ -31,3 +32,4 @@ class EZ_GUIFOUNDATION_DLL ezQtApplicationPanel : public ads::CDockWidget
 
   ezQtContainerWindow* m_pContainerWindow;
 };
+EZ_DECLARE_REFLECTABLE_TYPE(EZ_GUIFOUNDATION_DLL, ezQtApplicationPanel);
```

Comment:
Why is this reflected now ?

Reply:
Not strictly necessary but I fixed all cases where we stored a void pointer inside an ezVariant. By reflecting it you can now do dyncasts instead of just assuming it is the type you expected. Now I might have missed a few cases where we read the variants, so if you come across a .Get<void*>() or .ConvertTo<void*>() wrapped in a static_cast, please replace it by using the type directly.